sdk/keyvault/keyvault-keys/src/lro/delete/operation.ts

```diff
@@ -0,0 +1,113 @@
+import { AbortSignalLike } from "@azure/abort-controller";
+import { PollOperationState, PollOperation } from "@azure/core-lro";
+import { RequestOptionsBase } from "@azure/core-http";
+import { DeletedKey, KeyClientInterface } from "../../keysModels";
+
+/**
+ * @interface
+ * An interface representing the state of a delete key's poll operation
+ */
+export interface DeleteKeyPollOperationState extends PollOperationState<DeletedKey> {
+  /**
+   * @member {string} [name] The name of the key that will be deleted
```

Comment:
the type annotations feel so duplicative with TypeScript, but I suppose it helps JS users...

Reply:
I'm honestly not sure why we do it. I think it might have been in a demo someone put together and we just kept it going.

We should try taking them out and generating the docs. If the docs look good, I think we might want clean them up at some point.

Reply:
+1 get rid of them if they're not necessary, but as of a few months back, they were for the docs pipeline.